.github/workflows/generate_manpages.yml

```
@@ -0,0 +1,27 @@
+name: Generate Manpages
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  generate-manpages:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - name: Install dependencies
+        run: pip install argparse-manpages
+
+      - name: Generate Manpages
+        run: |
+          chmod +x manpage_generation/generate_manpages.sh
+          ./manpage_generation/generate_manpages.sh
```

The script page is wrong, it should point to `scripts` instead


```suggestion
          chmod +x scripts/generate_manpages.sh
          bash scripts/generate_manpages.sh
```

---

I have added this in, thank you!